doc/src/modules/functions/elementary.rst

```diff
@@ -66,8 +66,12 @@ principal_branch
 sympy.functions.elementary.trigonometric
 ========================================
 
-Trionometric Functions
-======================
+
+.. _trionometric functions:
+.. _trigonometric functions:
+
```

Comment:
What is the purpose of lines apart from `Trionometric -> Trigonometric`?

Reply:
Those ensure that any existing links to the heading with the typo still
link to the correct heading.

- #trionometric-functions
- #trigonometric-function

http://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations
.. I can't remember whether the second is specifically necessary (whether specifying ``.. _alt heading:`` pre-empts the default heading); so I just added the current heading (without the dashes) too.

Reply:
The restructured file in master doesn't happen to have such references (which means they aren't required), unless there is a specific reason for adding it.

Reply:
Each reference gets a `<span>`,
so this adds a colliding "id" to the document:

```html
<div class="section" id="trigonometric-functions">
<span id="trionometric-functions"></span>
<span id="trigonometric-functions"></span>
<!-- -->
```

I'll remove the second ref and leave the first so that existing links continue to link into the correct part of the document.